checkstyle-tester/my_check.xml

```diff
@@ -21,9 +21,11 @@
     <module name="TreeWalker">
          <!-- Example of sevntu.checkstyle Check usage -->
          <!-- <module name="NestedSwitchCheck"/> -->
-         
+
          <!-- Example of checkstyle Check usage -->
-         <module name="AbstractClassName"/>
+        <module name="ThrowsCount">
```

Comment:
No, comments above have incorrect indentation level, not the ``` <module name="ThrowsCount">```. I fixed the indentation level of comments.